platforms/jvm/language-java/src/main/java/org/gradle/api/tasks/javadoc/Javadoc.java

```diff
@@ -322,7 +329,18 @@ public boolean isVerbose() {
      */
     public void setVerbose(boolean verbose) {
         if (verbose) {
+            DeprecationLogger.deprecateMethod(Javadoc.class, "setVerbose(true)")
+                .withAdvice("Please use the getOptions().verbose() method instead. It may be necessary to cast the options to CoreJavadocOptions to access the replacement method.")
```

Comment:
Why would casting this be necessary? It looks like `MinimalJavadocOptions` has the method?

Reply:
Ah, you're right. I'll remove this.